src/puppy/platforms/win32/platform.nim

```diff
@@ -118,6 +118,44 @@ proc fetch*(req: Request): Response {.raises: [PuppyError].} =
         PuppyError, "WinHttpSendRequest error: " & $GetLastError()
```

Comment:
Since we're checking error and possibly retrying, we do not want to raise an exception yet. This should be removed to get windows passing the tests (https://github.com/treeform/puppy/runs/7869416921?check_suite_focus=true)

Reply:
It looks like this is probably fix needed to get things ready to merge.

Reply:
alright I'll remove it.